src/Mutator/Arithmetic/RoundingFamily.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator\Arithmetic;
+
+use Infection\Mutator\Util\Mutator;
+use PhpParser\Node;
+
+/**
+ * @internal
+ */
+final class RoundingFamily extends Mutator
```

Comment:
this one shouldn't be in this PR should it?

Reply:
It's the best example of how the new approach works, and it shouldn't be implemented with 3 classes like it was in another PR.

Of course, I can extract it, but this is extra work and I don't see why we should do it at this moment ;)